code/system/application/knowledge/src/linguistics/v2vcam/v2vcam.cpp

```diff
@@ -456,22 +485,46 @@ int32_t V2vCam::GetVehicleWidth() const
 
 int32_t V2vCam::GetLongitudinalAcc() const
 {
-  return m_longitudinalAcc;
+  int32_t scale = std::pow(10,1);
+  return static_cast<int32_t> (m_longitudinalAcc*scale);
 }
 
 int32_t V2vCam::GetLongitudinalAccConf() const
 {
-  return m_longitudinalAccConf;
+  int32_t scale = std::pow(10,1);
+  return static_cast<int32_t> (m_longitudinalAccConf*scale);
 }
 
 int32_t V2vCam::GetYawRateValue() const
 {
-  return m_yawRateValue;
+  int32_t scale = std::pow(10,2);
+  double conversion = opendlv::Constants::RAD2DEG;
+  return static_cast<int32_t> (m_yawRateValue*scale*conversion);
 }
 
 int32_t V2vCam::GetYawRateConfidence() const
 {
-  return m_yawRateConfidence;
+  double conversion = opendlv::Constants::RAD2DEG;
+  double val = m_yawRateConfidence * conversion;
+  if(m_yawRateConfidence < 0){
```

Comment:
There should be spaces around keywords and brackets.
if (a == b) {
} else if (a == c) {
} else {
}